cms-contentful/it/ContentfulCmsServiceIT.java

```diff
@@ -70,6 +70,14 @@ public void whenAskForNotExistingAssetContentThenNotPresent() throws Exception {
         assertThat(content).isEmpty();
     }
 
+    @Test
+    public void whenEntryTypeDoesNotExist_thenThrowException() throws Exception {
```

Comment:
I would expect that if you give a `CmsIdentifier` that doesn't exist, it will return `Optional.empty`, no matter which part didn't exist.
If the connection fails, the credentials are invalid, etc. Then an exception should be thrown.